### PR TITLE
refactor(all): update topic names

### PR DESCRIPTION
**Pull request by DavidFederl**
New Topics:
- `accelerater`/`g-value` -> `acceleration`
- `time` -> `timer`

### CODE REVIEW

File: cloud/app/src/main/java/de/ude/ies/elastic_ai/BallChallengeApplication.java [hunk 1]
```diff
@@ -17,11 +17,13 @@
 @SpringBootApplication
 public class BallChallengeApplication {
 
+    public static String HOST_IP;
```
Comment (glencoe) [inline cloud/app/src/main/java/de/ude/ies/elastic_ai/BallChallengeApplication.java:20]: what's host ip being used for now?

File: enV5/src/DataCollectionAdxl345b.c [hunk 2]
```diff
@@ -239,8 +245,8 @@ static char *collectSamples(void) {
 
 static void publishMeasurements(char *data) {
     if (strlen(data) > 0) {
-        char *topic = malloc(strlen("g-value") + 1);
-        strcpy(topic, "g-value");
+        char *topic = malloc(strlen(ACCELERATION_TOPIC) + 1);
```
Comment (glencoe) [inline enV5/src/DataCollectionAdxl345b.c:248]: Haha... That's a change i've been applying with my student in a fork as well :D

**Review by glencoe — APPROVED** [review]
Got only the single question about the host ip in the java class. Rest looks good to me. Thanks

File: cloud/app/src/main/java/de/ude/ies/elastic_ai/BallChallengeApplication.java (hunk 1 above)
Comment (DavidFederl) [reply cloud/app/src/main/java/de/ude/ies/elastic_ai/BallChallengeApplication.java:20]: > [!NOTE] 
> **TL;DR**: Currently an unused artifact from copy pasting monitor code!

To be honest. Currently inside the BallChallenge application for nothing. I want to add some additional features. Update the status message with the actual address to reach the webpage and the actual IP for external clients can't be determined from inside the container due to docker restrictions.
Beside this, I plan to move these value extraction to the application.properties file in combination with Beans (the actual SpringBoot way).
I hope that helps.

File: enV5/src/DataCollectionAdxl345b.c (hunk 2 above)
Comment (DavidFederl) [reply enV5/src/DataCollectionAdxl345b.c:248]: Boy-Scout-Rule in action!
